components/liveStreamBanner.tsx

```diff
@@ -45,45 +52,62 @@ export const LiveStreamBanner = () => {
       const event = res?.data
         .map((e) => new Event(e))
         .sort((a, z) => a.StartDateTime - z.StartDateTime)[0];
-      setEvent(event);
-
-      const startDateTime = dayjs(event.StartDateTime);
+      await setEvent(event);
 
+      const start = await dayjs(event.StartDateTime)
+      await setStartDateTime(start);
+      await setCountdownMins(start.diff(dayjs(), "minute"));
+      
       const isNow = dayjs(datetime).isBetween(
-        startDateTime,
+        start,
         dayjs(event.EndDateTime)
       );
+
       setIsLive(isNow);
     };
 
     fetchEvent();
+
+    const interval = setInterval(() => {
+      if (countdownMins >= -1) {
+        setCountdownMins(countdownMins => countdownMins - 1);
+      }
+    }, 60000);
+
+    return () => clearInterval(interval);
   }, []);
 
-  if (event.StartDateTime === undefined) return <></>;
+  useEffect(() => {
+    setCountdownText(countdownTextFormat(countdownMins));
+    setIsLive(countdownMins <= 0);
+  }, [countdownMins])
+
+  if (startDateTime === undefined) return <></>;
 
-  const startDateTime = dayjs(event.StartDateTime);
   const isSameDay = startDateTime.isSame(dayjs(), "day");
 
   if (isSameDay) {
     const liveText = "Streaming live now.";
-    const countdownText = `Airing ${startDateTime.fromNow()}. `;
     return (
+      <div className="w-full bg-gray-900">
       <a href="https://ssw.com.au/live">
+        
         <div
           className={classNames(
-            "h-liveStream bg-gray-900 bg-right-top bg-no-repeat p-3 uppercase",
+            "bg-gray-900 bg-right-top bg-no-repeat p-5 uppercase mx-auto max-w-7xl px-6 sm:px-8",
             isLive ? "bg-live-banner-live" : "bg-live-banner-wait"
           )}
         >
-          <h1 className="text-xl font-light text-gray-300">{event.Title}</h1>
-          <p className=" text-xs text-white">
+          <h1 className="m-0 py-0 text-xl font-light text-gray-300">{event.Title}</h1>
+          <p className="py-0 text-xs text-white">
             <span className="text-sswRed">
               {isLive ? liveText : countdownText}
             </span>
             {!isLive && scheduledTimeText(startDateTime)} #NetUG
```

Comment:
Can this be sourced from somewhere instead of being hard coded?
I know we sometimes livestream the #NCCG and it would be good to use their hashtag when appropriate

Reply:
As per our conversation, I raised issue #236